### ISSUE TITLE
Enable "Hide EResources To Which I don't have access" as default

### ISSUE THREAD

**Issue by dmcmpbll** [CARLI/vufind#325]
5/9/2018 - During the New VuFind Overview Webinar Q &A, attendees inquired whether it might be possible to make the “Hide eResources to which I don’t have access” limit be set as “checked” by default

**Comment by Rolfing-Library** [comment]
Yes! TIU was one of those requesting this change. It would be very helpful so that our users are not confused by/wasting time trying to access these resources.

**Comment by gibsonjc** [comment]
One can easily and safely prelimit to apply the filter for "Hide eResources to which I don't have access" in the I-Share union catalog using a custom search box that can be embeded into any webpage. 

Below is a very basic example search box.  To modify this for your library, replace the "xxx" in "vf-**xxx**" in the "form action" with your library's three-letter I-Share code in lower case:

```
<form action="https://i-share.carli.illinois.edu/all/vf-xxx/Search/Results" class="search" method="get" name="searchForm" target="vufind">
<fieldset>
<label for="lookFor"><strong>Search I-Share but not eResources I don't have access to: </strong></label>
<input name="start_over" type="hidden" value="1" />
<input id="lookFor" name="lookfor" size="20" type="text" value="" />
<label class="hidden" for="searchType">Search Type</label>
<select id="searchType" name="type">
<option value="allfields">All Fields</option>
<option value="title">Title</option>
<option value="author">Author</option>
<option value="subject">Subject</option>
<option value="isn">ISBN/ISSN</option>
<option value="JournalTitle">Journal Title</option>
</select> 
<input name="filter[]" type="hidden" value="building:*" /> <input name="submit" type="submit" value="Find" />
</fieldset>
</form>
```

(You can also change the "label" to be anything you like.)

This basic box looks like this:
![screen shot 2018-08-16 at 9 26 03 am](https://user-images.githubusercontent.com/17598357/44214742-764f1400-a136-11e8-8d5d-1db795d1f3d2.png)

A web developer can adjust it to customize the look and feel as needed.

**Comment by gibsonjc** [comment]
Instead of a search box, you can also create a pre-limited URL to I-Share for in a link on a website.  Replace the xxx in vf-xxx with your library's three-letter I-Share code in lowercase:

`https://i-share.carli.illinois.edu/all/vf-xxx/Search/Results?filter[]=building%3A*`

The effect is that the user by-passes the I-Share welcome page and lands immediately on a Results page, where the result list is "everything except eresources to which I don't have access." This could be disconcerting for users. The checkbox would remain checked for subsequent searches unless the user unchecks it, but only for users who used that link method to connect.

For iit for example:
[Search I-Share for IIT users hiding e-resources they don't have access to](https://i-share.carli.illinois.edu/all/vf-iit/Search/Results?filter[]=building%3A*)
